pressbooks/pressbooks
# Add LaTeX button to visual editor

This PR fixes https://github.com/pressbooks/ideas/issues/383 by adding a LaTeX button to the visual editor.

To test:
1. Check out this branch and open the chapter editor.
2. Select some text and click the new LaTeX button in the visual editor. The text should be automatically wrapped in [latex] [/latex] shortcodes
3. Press the LaTeX button when no text is selected: A 'LaTeX Content' prompt should appear with a default value of 'Enter your LaTeX content here'. When you enter content and click ok, the content should be added to the visual editor with [latex] [/latex] shortcodes wrapped around it.
